api/auth/get_token.go

```diff
@@ -69,6 +79,32 @@ func GetAuthToken(c *gin.Context) {
 	// capture the OAuth state if present
 	oAuthState := c.Request.FormValue("state")
 
+	// handle scm setup events
+	// setup_action==install represents the GitHub App installation callback redirect
+	if c.Request.FormValue("setup_action") == constants.AppInstallSetupActionInstall {
+		installID, err := strconv.ParseInt(c.Request.FormValue("installation_id"), 10, 0)
```

Comment:
we need to add installation_id to swagger annotation?

Reply:
done